### ISSUE TITLE
Accessibility: Empty button

### ISSUE THREAD

**Issue by gabek** [owncast/owncast#2625]
### Share your bug report, feature request, or comment.

Not sure what this is referring to, why it's there, or who is responsible.

> What It Means
A button is empty or has no value text.
Why It Matters
When navigating to a button, descriptive text must be presented to screen reader users to indicate the function of the button.
How to Fix It
Place text content within the <button> element or give the <input> element a value attribute.
The Algorithm... in English
A <button> element is present that contains no text content (or alternative text), or an <input type="submit">, <input type="button">, or <input type="reset"> has an empty or missing value attribute.

![image](https://user-images.githubusercontent.com/414923/214401452-0ddd21cb-e5cf-4ecd-bcfc-727e5553fbd4.png)

See #1826 for more accessibility discussion.

**Comment by gabek** [comment]
I haven't been able to track this down. It's probably not a big deal, but if anybody else wants to look before I close it please feel free.

**Comment by gabek** [comment]
Figured it out, it's a VideoJS thing that only happens when playback is taking place. Closing since I'm not looking to address VideoJS accessibility.
